informers.go

```diff
@@ -62,17 +70,25 @@ func updateMachineTypes(virtCli kubecli.KubevirtClient) error {
 			
 			// add label to running VMs that a restart is required for change to take place
 			if vm.Status.Ready {
-				addWarningLabel(virtCli, &vm)
+				err = addWarningLabel(virtCli, &vm)
+				if err != nil {
+					return err
+				}
 			}
 		}
 	}
 	return nil
 }
 
-func addWarningLabel (virtCli kubecli.KubevirtClient, vm *k6tv1.VirtualMachine) {
+func addWarningLabel (virtCli kubecli.KubevirtClient, vm *k6tv1.VirtualMachine) error {
 	addLabel := fmt.Sprint(`{"metadata": {"labels": {"restart-vm-required": "true"}}}}}`)
-	virtCli.VirtualMachine(vm.Namespace).Patch(vm.Name, types.StrategicMergePatchType, []byte(addLabel), &k8sv1.PatchOptions{})
-	needsRestart++
+	virtCli.VirtualMachineInstance(vm.Namespace).Patch(context.Background(), vm.Name, types.StrategicMergePatchType, []byte(addLabel), &k8sv1.PatchOptions{})
+	if err != nil {
```

Comment:
Where is the `err` variable declared / initialized?
I think you need `err := virtCli.VirtualMachineInstance(vm.Namespace).Patch(....)`